lib/plug/builder.ex

```diff
@@ -119,13 +120,14 @@ defmodule Plug.Builder do
 
   @doc false
   defmacro __before_compile__(env) do
```

Comment:
Here is the env... just pass it forward up to the point we generate the quote. :)